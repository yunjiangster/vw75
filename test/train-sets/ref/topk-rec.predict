2.000016  J-A 
2.999566  J-J 

1.000159  M-A 
2.999230  M-M 

2.000115  A-J 
2.999882  A-A 

