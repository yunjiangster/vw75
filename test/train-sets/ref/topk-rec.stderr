Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = topk.predict
using no cache
Reading datafile = train-sets/topk.vw
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.000000   0.000000            1         1.0   3.0000   2.9996        4
0.000000   0.000000            2         2.0   0.0000   0.0005        4
0.000000   0.000000            3         3.0   2.0000   2.0000        4
0.000000   0.000000            4         4.0  unknown   0.0000        1
0.000000   0.000000            5         5.0   0.0000   0.0004        4
0.000000   0.000001            6         6.0   3.0000   2.9992        4
0.000000   0.000000            7         7.0   1.0000   1.0002        4
0.000000   0.000000            8         8.0  unknown   0.0000        1
0.000000   0.000000            9         9.0   2.0000   2.0001        4
0.000000   0.000000           10        10.0   1.0000   1.0001        4
0.000000   0.000000           11        11.0   3.0000   2.9999        4
0.000000   0.000000           12        12.0  unknown   0.0000        1

finished run
number of examples per pass = 12
passes used = 1
weighted example sum = 12
weighted label sum = 0
average loss = 1.09109e-07
best constant = 0
total feature number = 39
